.github/workflows/e2e-tests.yml

```diff
@@ -25,14 +25,13 @@ jobs:
       - uses: actions/checkout@v2
       - id: create-matrix
         run: |
-          MATRIX='[{"group":"pre-connection","tests":"pre-connection"},{"group":"connection","tests":"connection"},{"group":"post-connection","tests":"post-connection"}]'
+          MATRIX='[{"group":"pre-connection","tests":"pre-connection"},{"group":"connection","tests":"specs/connection"},{"group":"post-connection","tests":"post-connection"}]'
```

Comment:
The issue was introduced after my comment here: https://github.com/Automattic/jetpack/pull/21848#discussion_r755125583

What do you think about updating all other group instances to include `specs/*` in the path?

Reply:
Good point. I made the change in [a4665e4](https://github.com/Automattic/jetpack/pull/21999/commits/a4665e4ed5a0b3971e2b2eb70aa9ae6d5a56040d)